### PR TITLE
feat: Buttons

### PR DESCRIPTION
**Pull request by gulam-mujtaba**
### Fourth Component Added to Component Library.

##Buttons

**Features Added:**

1. Primary Buttons - basic, outline, link, icon.
2. Secondary Buttons - basic, outline, link, icon.
3. Different Sized buttons.
4. Disabled Buttons.
5. Floating Action Button

**Screenshots**
buttons ss1
b

### CODE REVIEW

File: components/badges.css [hunk 1]
```diff
@@ -0,0 +1,151 @@
+@import url(/main.css);
+
+.main-container {
+  display: flex;
+  gap: auto;
+}
+
+.sidebar {
+  height: 900px;
```
Comment (zabihshaik) [inline components/badges.css:9]: Better to use rems instead of px

File: components/badges.css [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+}
+
+li a {
+  font-size: 1.4rem;
+  color: white;
+  height: 3rem;
+  text-align: center;
+  margin-top: 18px;
+  display: flex;
+  margin-left: 30px;
```
Comment (zabihshaik) [inline components/badges.css:30]: Better to use rems instead of px, and shorthand property for margin

File: components/badges.css [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+.cart-icon {
+  color: black;
+  padding: 1rem;
+  padding-left: 2rem;
+  position: absolute;
+}
+
+.cart-badge {
+  background-color: rgb(44, 44, 131);
```
Comment (zabihshaik) [inline components/badges.css:77]: Better to use variable for this color as it is being used repeatedly.

**Review by zabihshaik — COMMENTED** [review]
Everything looks good. Please check the comments.

File: components/badges.css (hunk 1 above)
Comment (gulam-mujtaba) [reply components/badges.css:9]: Sure

File: components/badges.css (hunk 3 above)
Comment (gulam-mujtaba) [reply components/badges.css:77]: Ok, Noted. Thanks for pointing out.

File: components/badges.css (hunk 2 above)
Comment (gulam-mujtaba) [reply components/badges.css:30]: Noted.

File: components/alert.css [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  background-color: white;
+  border-radius: 5px;
+}
+
+h6 {
+  padding: 1rem 0.5rem;
+}
+.alert-primary {
+  display: flex;
+  width: 600px;
```
Comment (krishanu2911) [inline components/alert.css:16]: just a suggestion if it's a greater value then it is preferred to use rems over px

File: components/alert.css [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+  padding: 0.5rem;
+  margin: 10px 0;
+  background-color: #c2c2c2;
```
Comment (krishanu2911) [inline components/alert.css:19]: you can use CSS variable it will be convenient for your usage and also recommended

File: components/alert.css [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+  margin: 10px 0;
+  background-color: #ebe8e8;
+  border-radius: 5px;
+  font-weight: medium;
+}
+
+.alert-success,
+.alert-success-icon {
+  display: flex;
+  width: 600px;
```
Comment (krishanu2911) [inline components/alert.css:37]: same here

File: components/buttons.css [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+  border-radius: 5px;
+}
+
+button:hover {
+  cursor: pointer;
+}
+
+h2,
+h4 {
+  padding: 15px 10px;
```
Comment (krishanu2911) [inline components/buttons.css:17]: preffer rems if possible

File: components/buttons.css [hunk 8]
```diff
@@ -0,0 +1,177 @@
...
+.btn-secondary-simple,
+.btn-secondary-outline,
+.btn-small,
+.btn-large,
+.btn-primary-icon,
+.btn-secondary-icon,
+.btn-disabled,
+.btn-disabled-2,
+.link-btn-primary,
+.link-btn-secondary {
```
Comment (krishanu2911) [inline components/buttons.css:36]: instead of using declaring for all the classes define a common class and use it wherever you need

**Review by krishanu2911 — COMMENTED** [review]
Well written code just added a few suggestions below 
that try to use CSS variable wherever possible and prefer rem over px in larger values.
else everything is good to go!!

**Pull request merged**